### PR TITLE
feat(route): add 湖北大学资源环境学院

### LINKED ISSUE #14948: 湖北大学官网新闻更新
### 类型

大学通知

### 网站地址

https://www.hubu.edu.cn/index/tzgg.htm

### 网站描述

湖北大学的新闻发布网站

### 需要生成什么内容？

获取湖北大学发布的新消息

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### LINKED ISSUE #14949: 湖北大学资环学院的新闻通知
### 类型

大学通知

### 网站地址

https://zhxy.hubu.edu.cn/index/xwdt.htm

### 网站描述

湖北大学资环学院的新闻

### 需要生成什么内容？

获取湖北大学资环学院的新闻更新

### 额外描述

对计算机编程一窍不通，请见谅，只能求助

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by nczitzk**
## Involved Issue / 该 PR 相关 Issue

Close #14948
Close #14949

## Example for the Proposed Route(s) / 路由地址示例

```routes
/hubu/www/index/tzgg
/hubu/zhxy/index/tzgg
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/hubu/index.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        supportRadar: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            title: '通知公告',
+            source: ['hubu.edu.cn/index/tzgg.htm'],
+            target: '/hubu/www/index/tzgg',
```
Comment (TonyRL) [inline lib/routes/hubu/index.ts:116]:
```suggestion
            target: '/www/index/tzgg',
```

File: lib/routes/hubu/index.ts [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        },
+        {
+            title: '学术预告',
+            source: ['hubu.edu.cn/index/xsyg.htm'],
+            target: '/hubu/www/index/xsyg',
```
Comment (TonyRL) [inline lib/routes/hubu/index.ts:121]:
```suggestion
            target: '/www/index/xsyg',
```

File: lib/routes/hubu/zhxy.ts [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+        },
+        {
+            title: '党群工作 - 党建工作',
+            source: ['zhxy.hubu.edu.cn/dqgz/djgz/jgdj.htm'],
+            target: '/hubu/zhxy/dqgz/djgz/jgdj',
+        },
+        {
+            title: '党群工作 - 工会工作',
+            source: ['zhxy.hubu.edu.cn/dqgz/ghgon.htm'],
+            target: '/hubu/zhxy/dqgz/ghgon',
```
Comment (TonyRL) [inline lib/routes/hubu/zhxy.ts:225]:
```suggestion
            target: '/zhxy/index/tzgg',
        },
        {
            title: '新闻动态',
            source: ['zhxy.hubu.edu.cn/index/xwdt.htm'],
            target: '/zhxy/index/xwdt',
        },
        {
            title: '人才培养',
            source: ['zhxy.hubu.edu.cn/rcpy.htm'],
            target: '/zhxy/rcpy',
        },
        {
            title: '人才培养 - 本科生教育',
            source: ['zhxy.hubu.edu.cn/rcpy/bksjy.htm'],
            target: '/zhxy/rcpy/bksjy',
        },
        {
            title: '人才培养 - 研究生教育',
            source: ['zhxy.hubu.edu.cn/rcpy/yjsjy.htm'],
            target: '/zhxy/rcpy/yjsjy',
        },
        {
            title: '人才培养 - 招生与就业',
            source: ['zhxy.hubu.edu.cn/rcpy/zsyjy/zsxx.htm'],
            target: '/hubu/zhxy/rcpy/zsyjy/zsxx',
        },
        {
            title: '学科建设',
            source: ['zhxy.hubu.edu.cn/xkjianshe/zdxk.htm'],
            target: '/zhxy/xkjianshe/zdxk',
        },
        {
            title: '学科建设 - 重点学科',
            source: ['zhxy.hubu.edu.cn/xkjianshe/zdxk.htm'],
            target: '/zhxy/xkjianshe/zdxk',
        },
        {
            title: '学科建设 - 硕士点',
            source: ['zhxy.hubu.edu.cn/xkjianshe/ssd.htm'],
            target: '/zhxy/xkjianshe/ssd',
        },
        {
            title: '学科建设 - 博士点',
            source: ['zhxy.hubu.edu.cn/xkjianshe/bsd.htm'],
            target: '/zhxy/xkjianshe/bsd',
        },
        {
            title: '科研服务',
            source: ['zhxy.hubu.edu.cn/kyfw.htm'],
            target: '/zhxy/kyfw',
        },
        {
            title: '科研服务 - 科研动态',
            source: ['zhxy.hubu.edu.cn/kyfw/kydongt.htm'],
            target: '/zhxy/kyfw/kydongt',
        },
        {
            title: '科研服务 - 学术交流',
            source: ['zhxy.hubu.edu.cn/kyfw/xsjl.htm'],
            target: '/zhxy/kyfw/xsjl',
        },
        {
            title: '科研服务 - 科研平台',
            source: ['zhxy.hubu.edu.cn/kyfw/keyapt.htm'],
            target: '/zhxy/kyfw/keyapt',
        },
        {
            title: '科研服务 - 社会服务',
            source: ['zhxy.hubu.edu.cn/kyfw/shfuw.htm'],
            target: '/zhxy/kyfw/shfuw',
        },
        {
            title: '党群工作',
            source: ['zhxy.hubu.edu.cn/dqgz.htm'],
            target: '/zhxy/dqgz',
        },
        {
            title: '党群工作 - 党建工作',
            source: ['zhxy.hubu.edu.cn/dqgz/djgz/jgdj.htm'],
            target: '/zhxy/dqgz/djgz/jgdj',
        },
        {
            title: '党群工作 - 工会工作',
            source: ['zhxy.hubu.edu.cn/dqgz/ghgon.htm'],
            target: '/zhxy/dqgz/ghgon',
```

File: lib/routes/hubu/index.ts (hunk 1 above)
Comment (nczitzk) [reply lib/routes/hubu/index.ts:116]: 😊

**Pull request merged**